File: Android/app/src/androidTest/java/jp/co/riso/smartdeviceapp/AppConstantsTest.java
package jp.co.riso.smartdeviceapp;

import junit.framework.TestCase;

public class AppConstantsTest extends TestCase {
    
    protected void setUp() throws Exception {
        super.setUp();
    }
    
    protected void tearDown() throws Exception {
        super.tearDown();
    }
    
    //================================================================================
    // Tests - constructors
    //================================================================================
    
    public void testConstructor() {
        AppConstants appConstants = new AppConstants();
        assertNotNull(appConstants);
    }
}

File: Android/app/src/androidTest/java/jp/co/riso/smartdeviceapp/AppConstantsTest.kt
package jp.co.riso.smartdeviceapp

import kotlin.Throws
import jp.co.riso.smartdeviceapp.AppConstants
import junit.framework.TestCase
import java.lang.Exception

class AppConstantsTest : TestCase() {
    @Throws(Exception::class)
    override fun setUp() {
        super.setUp()
    }

    @Throws(Exception::class)
    override fun tearDown() {
        super.tearDown()
    }

    //================================================================================
    // Tests - constructors
    //================================================================================
    fun testConstructor() {
        val appConstants = AppConstants
        assertNotNull(appConstants)
    }
}